### DISCUSSION TITLE
unable to access the security onion console when security onion is installed in AWS ubutnu ec2 instance

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [Security-Onion-Solutions/securityonion#11493] [General] [answered]
Hi,  I have installed security onion in AWS instance of Ubuntu version 20.04  for requirement 4vcpu, 16 ram and 200 GB storage space.  sudo bash so-setup-network when i execute this command, at this point 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/a4090a79-f63f-4474-8e87-ce498af68059)

if i give localhost as host name its not taking.. and i tried of giving 0.0.0.0 so that i can access through any ip address but its not taking. at last it gives access URL as private IP address even though i didn't mentioned the private address anywhere in network setup its giving private IP address as web access URL, May be because of this reason  i'm unable access the console of security onion 

can you help me please!

**Comment by TOoSmOotH** [comment]
Are you using the official AMI from the marketplace?

**Reply by sandraimmaculate** [reply]
no

**Reply by sandraimmaculate** [reply]
im using ubuntu version 20.04

**Reply by sandraimmaculate** [reply]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/52be77a1-ac39-4f4a-a8a4-17fe61642766)
im using this as ami. Because i want to setup everything manually so that i will get to know more about security onion

**Reply by TOoSmOotH** [reply, marked as answer]
The AMI has differences in the setup that account for the fact that it is running in AWS. You would need to look through the source code to determine those differences and change the code accordingly if you do not want to use the AMI.

**Reply by hs3387** [reply]
I'm using the AMI and I still cannot access it?
I can hit the page from other ec2's inside my VPC but from the internet I can't 
(it's not a security group issue - I checked, I can send and receive traffic on all ports from the internet)

**Reply by TOoSmOotH** [reply]
Did you allow access to your ip address as an analyst role?
